### PR TITLE
Add queries and screenshots

### PR DESCRIPTION
**Pull request by dodoburner**
In this pull request, I have practiced using transactions to update and delete records. I also used my knowledge of aggregate functions and GROUP BY to answer analytical questions. 
I have provided screenshots in the screenshots folder.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-    id INT GENERATED ALWAYS AS IDENTITY, 
+    id INT, 
     name TEXT,
     date_of_birth DATE, 
     escape_attempts INT, 
     neutered BOOLEAN, 
     weight_kg DECIMAL,
+    species TEXT,
     PRIMARY KEY(id)
 );
```
Comment (sja-thedude) [inline schema.sql:11]: - [ ] Kindly make sure to add the query to alter the table. You have made changes to the existing one. You should add the query below the table as  `ALTER TABLE animals ADD species varchar(255);`

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,13 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+DELETE FROM animals WHERE date_of_birth < '01-Jan-2020';
+UPDATE animals SET weight_kg = weight_kg * -1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure to add  all queries for all the transactions.
Currently, you have only added the last transaction and also its not complete.
Transaction starts with 
`BEGIN;`
Example: (for the code you wrote)-
```sql
BEGIN;

    DELETE FROM animals WHERE  date_of_birth >= '2022/01/01' ;
    SAVEPOINT SP1;

    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 

    COMMIT;
```

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @dodoburner, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow

## Required 
- [ ] Kindly make sure to add screenshots for all queries in the pull request.
- [ ] Kindly update your readme file with correct details.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @dodoburner 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing all the changes  by the previous reviewer ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**